### PR TITLE
AO3-5961 Stop rails logger from logging so many things below level "error"

### PR DESCRIPTION
**Pull request by saltlas**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5961

## Purpose

What does this PR do?
Removes all instances of rails logger that use "warn" and below

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

The deleted messages do not appear.

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)? salt, they/them

### CODE REVIEW

**Comment by saltlas** [comment]
ah, conflicts. will fix those soon.

**Review by sarken — COMMENTED** [review]
Looks good to me, but @zz9pzza, do you need any of these for any reason? If not, feel free to approve!

File: app/models/download_writer.rb [hunk 1]
```diff
@@ -51,9 +51,6 @@ def generate_ebook_download
     cmds.each do |cmd|
       exit_status = nil
       Open3.popen3(*cmd) { |_stdin, _stdout, _stderr, wait_thread| exit_status = wait_thread.value }
-      unless exit_status
-        Rails.logger.debug "Download generation failed: " + cmd.to_s
```
Comment (sarken) [inline app/models/download_writer.rb:55]: Can we keep this but change the level to `warn`?

**Review by sarken — CHANGES_REQUESTED** [review]
(I spoke to @zz9pzza and am passing on his comments because he's a bit busy.) 

Can we also update `config.log_level` to `:error` in https://github.com/otwcode/otwarchive/blob/f7cd4fc80c1482e12d216422759b3eee670b45fb/config/environments/production.rb#L30

**Review by sarken — APPROVED** [review]
Thanks!

**Pull request merged**